src/views/SignIn.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+    <div class="row">
+  <div class="signIn w-100">
+    <form>
+      <b-col>
+        <b-form-group id="input-group-first-name" label="Your Firstname:" label-for="input-first-name">
+          <b-form-input
+            id="input-first-name"
+            class="w-100 input-styling"
+            type="text"
+            v-model="signIn.firstName"
+            required
+            placeholder="Max">
+          </b-form-input>
+        </b-form-group>
+      </b-col>
+      <b-col>
+      <b-form-group id="input-group-last-name" label="Your Lastname:" label-for="input-last-name">
+        <b-form-input
+          id="input-last-name"
+          type="text"
+          class="w-100 input-styling"
+          v-model="signIn.lastName"
+          required
+          placeholder="Mustermann">
+        </b-form-input>
+      </b-form-group>
+      </b-col>
+      <b-col>
+      <b-form-group id="input-group-initials" label="Your Initials:" label-for="input-initials">
+        <b-form-input
+          id="input-initials"
+          type="text"
+          class="w-100 input-styling"
+          v-model="signIn.initials"
+          required
+          placeholder="mmn"></b-form-input>
+      </b-form-group>
```

Comment:
Hier hab ich euch einen Medium Artikel:
https://medium.com/by-digiti/designing-the-perfect-input-field-aefb59bdd085
Laut dem würde man:


```suggestion
        <b-form-group id="input-group-first-name" label="First name" label-for="input-first-name">
          <b-form-input
            id="input-first-name"
            class="w-100 input-styling"
            type="text"
            v-model="signIn.firstName"
            required>
          </b-form-input>
        </b-form-group>
      </b-col>
      <b-col>
      <b-form-group id="input-group-last-name" label="Last name" label-for="input-last-name">
        <b-form-input
          id="input-last-name"
          type="text"
          class="input-styling"
          v-model="signIn.lastName"
          required>
        </b-form-input>
      </b-form-group>
      </b-col>
      <b-col>
      <b-form-group id="input-group-initials" label="Initials" label-for="input-initials">
        <b-form-input
          id="input-initials"
          type="text"
          class="w-100 input-styling"
          v-model="signIn.initials"
          required></b-form-input>
      </b-form-group>
```

Reply:
Cooler Artikel danke.